dev/idm-am-common-user/development/README.md

```diff
@@ -0,0 +1,249 @@
+You have two choices for running this development environment - within Minikube or within GKE. Both have advantages and disadvantages.
+
+Minikube:
+ - No cost (assuming you have the compute resources necessary on your own machine)
+ - No network dependency - can work offline
+ - No credentials needed for interacting with GKE or GCR
+ - Faster deployment of changes
+
+GKE:
+ - Publicly available
+ - Uses real HTTPS certificate instead of self-signed
+ - Less environment to setup
+ - Uses no resources on your own machine
+
+Choose whichever option works best for you. Steps for setting up either are described below:
+
+## Initial Preparation
+
+Have the following installed locally:
+
+- Docker https://docs.docker.com/install/
+- kubectl https://kubernetes.io/docs/tasks/tools/install-kubectl/
+- Helm https://github.com/helm/helm#install
+- Skaffold https://github.com/GoogleContainerTools/skaffold#installation
+
+Clone the [forgeops](https://github.com/ForgeRock/forgeops/) git repository at the same level as this forgeops-init folder. So it should be available at ../../../../forgeops (relative to this folder). This is necessary for the "helm" symbolic link to map to the helm charts published in forgeops.
+
+## Using Minikube
+
+Install Minikube: https://kubernetes.io/docs/tasks/tools/install-minikube/
+
+1. Prepare your minikube VM. This is only needed the first time the VM is created; if you restart your host or your VM, you do not need to repeat this step.
+
+    This creates the minikube VM, enables the ingress addon, adds a new namespace to your kubectl config, initializes the helm tiller, and installs the cert-manager chart:
+
+    ```
+    minikube start --insecure-registry 10.0.0.0/24 --memory 4096 && \
+    minikube addons enable ingress && \
+    kubectl config set-context sample-context --namespace=sample --cluster=minikube --user=minikube && \
+    sleep 2 && \
+    helm init --wait && \
+    helm upgrade -i cert-manager --namespace kube-system stable/cert-manager
+    ```
+
+2. You will need to run these commands every time the VM starts (after first step as well as after every VM reboot).
+
+    These commands fix a bug in minikube related to loopback networking, prepare your Docker environment to point to the minikube Docker service, and instructs kubectl to use the proper namespace for this sample.
+
+    ```
+    minikube ssh "sudo ip link set docker0 promisc on" && \
+    eval $(minikube docker-env) && \
+    kubectl config use-context sample-context
+    ```
+
+3. Use skaffold to build the images and deploy the helm charts:
+
+    ```
+    skaffold dev
+    ```
+
+    This will build the docker images and incorporate them into the helm templates, followed by managing the release of the charts. Any changes made to the configuration files for each docker image (found under ../amster, ../idm, ../rs) will be watched by skaffold, and will result in an automatic rebuild of the associated image followed by a redeployment into the cluster.
```

Comment:
Lingering reference to "rs" here. Given that was removed (for the time being) probably want to scrub all references to it.